bciers/apps/registration/app/components/operations/Operations.tsx

```diff
@@ -0,0 +1,74 @@
+import { GridRowsProp } from "@mui/x-data-grid";
+
+import buildQueryParams from "@/app/utils/buildQueryParams";
+import OperationDataGrid from "./OperationDataGrid";
+import { FrontEndRoles } from "@/app/utils/enums";
+import { OperationRow, OperationsSearchParams } from "./types";
+import { actionHandler } from "@/app/utils/actions";
+
+export const formatOperationRows = (rows: GridRowsProp) => {
+  if (!rows) {
+    return;
+  }
+  return rows.map(({ id, operator, name, bcghg_id, type }) => {
+    return {
+      id,
+      name,
+      bcghg_id,
+      operator: operator,
+      type,
+    };
+  });
+};
```

Comment:
Is this function even necessary? It doesn't look like it's doing anything useful.